modules/core/04-channel/keeper/packet.go

```diff
@@ -283,9 +283,9 @@ func (k *Keeper) RecvPacketV2(
 		return errorsmod.Wrap(err, "couldn't verify counterparty packet commitment")
 	}
 
-	//if err := k.applyReplayProtection(ctx, packet, channel); err != nil {
+	// if err := k.applyReplayProtection(ctx, packet, channel); err != nil {
```

Comment:
#7250

Reply:
do you know why this is commented out?

Reply:
I think it may have been commented out to keep diff low, it'll require a duplicate v2 version of all the functions down that code path.

I think we can probably come up with a cleaner solution in the end where the v2 code path is the only path anyway.

Reply:
also just realised this entire fn doesn't need to exist, channelKeeper.RecvPacketV2 was the impl that existed on the port router branch, we can remove this since the v2 flow will always go through the PacketServerKeeper